packages/frontend-test-utils/src/app/renderInTestApp.tsx

```diff
@@ -34,5 +63,26 @@ export function renderInTestApp(element: JSX.Element) {
     factory: () => ({ element }),
   });
   const tester = createExtensionTester(extension);
+
+  if (options?.mountedRoutes) {
+    for (const [path, routeRef] of Object.entries(options.mountedRoutes)) {
+      // TODO(Rugvip): add support for external route refs
```

Comment:
pretty common use case right? well, maybe less so with the default targets feature